website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,178 @@
+# Overview - Single API version operator
+
+This guide walks through the steps required to migrate an operator from an Operator SDK project layout to a Kubebuilder project layout.
+
+The document considers [memcached operator][memcached-operator] as an example to describe the process of migrating an operator built using Operator SDK to a runnable Kubebuilder project.
+
+## Prerequisites
+- [git][git_tool].
+- [go][go_tool] version 17.03+.
+- [kubectl][kubectl_tool] version v1.11.3+.
+- [kustomize][kustomize_tool] v3.1.0+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+## Install the Operator SDK CLI
+
+Follow the steps in the [installation guide][install_guide] to learn how to install the Operator SDK CLI tool.
+
+## Create a new project
+
+Use `operator-sdk init` command to create a new project:
+
+```sh
+$ mkdir $HOME/projects/memcached-operator
+$ cd $HOME/projects/memcached-operator
+$ operator-sdk init --domain example.com --license apache2 --owner "The Memcached Operator authors" --repo github.com/example-inc/memcached-operator
+$ cd memcached-operator
+```
+
+Make sure to activate the go module support by running `$ export GO111MODULE=on`.
+
+## Create an API
+
+Create a new API and a new kind (CRD) for the operator.
+
+In case of memcached operator, a new API named `cache/v1aplha1` (group/version) and a new kind `Memcached` is created using the following command.
+
+`operator-sdk api --group cache --version v1alpha1 --kind Memcached`
+
+Press `y` when asked for creating resource and controller. This will scaffold the project and create the files `api/<version>/<kind>_types.go` and `controller/<kind>_types.go`.
+
+For memcached operator project, `api/v1alpha1/memcached_types.go` and `controller/memcached_controller.go` are generated.
+
+```sh
+Create Resource [y/n]
+y
+Create Controller [y/n]
+y
+Writing scaffold for you to edit...
+api/v1alpha1/memcached_types.go
+controllers/memcached_controller.go
+Running make:
+$ make
+go: creating new go.mod: module tmp
+go: finding sigs.k8s.io v0.2.5
+go: finding sigs.k8s.io/controller-tools/cmd v0.2.5
+go: finding sigs.k8s.io/controller-tools/cmd/controller-gen v0.2.5
+/Users/admin/go/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
+go fmt ./...
+go vet ./...
```

Comment:
I think it is not required. 
I mean, why we need to add this output? What info/problem will be solved with? Is not better we try to be less verbose here to make the things simple?